src/components/NavMobile/NavMobile.scss

```diff
@@ -0,0 +1,332 @@
+    align-items: center;
+    justify-content: space-between;
+    overflow: hidden;
+    height: 5rem;
+    padding: 1rem 2rem;
+
+    &__header {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: center;
+    }
+
+    &__logo {
+      width: 5rem;
+      height: 5rem;
+      margin-left: -0.5rem;
+      fill: $primary-1;
+    }
+
+    &__title {
+      display: block;
+      margin: 0;
+      padding: 0;
+      color: $dark-grey;
+      font-size: 1.25rem;
+      font-family: $lora;
+
+      &--sm {
+        font-size: 0.75rem;
+        font-family: $poppins;
+      }
+    }
+
+    &__hamburger {
+      width: 1.5rem;
+    }
+
+    &__listItem {
+      padding: 0;
+    }
+
+    &__sidebar {
+      position: fixed;
+      z-index: 999;
+      display: none;
+      flex-direction: column;
+      overflow: hidden;
+      overflow-y: scroll;
+      width: 35%;
+      height: 100%;
+      padding: 2rem;
+      background-color: $white;
+      filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+
+      &--active {
+        @keyframes slideIn {
+          from {
+            right: -35rem;
+          }
+
+          to {
+            right: 0;
+          }
+        }
+
+        @extend .nav-mobile__sidebar;
+        top: 0;
+        right: 0;
+        display: flex;
+        animation: slideIn 200ms ease-out;
+      }
+
+      &--inactive {
+        @keyframes slideOut {
+          from {
+            right: 0;
+          }
+
+          to {
+            right: -35rem;
+          }
+        }
+
+        @extend .nav-mobile__sidebar;
+        position: fixed;
+        top: 0;
+        right: -35rem;
+        display: flex;
+        animation: slideOut 200ms ease-out;
+      }
+
+      &--items {
+        display: flex;
+        flex-direction: column;
+        margin: 1rem 0;
+        padding: 0;
+        list-style-type: none;
+      }
+
+      &--item {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
+        padding: 1rem 0;
+        font-weight: 600;
+      }
+
+      &--carrot {
+        margin-right: 1rem;
+
+        &--active {
+          @keyframes rotateCarrot {
+            0% {
+              transform: rotate(0deg);
+            }
+
+            100% {
+              transform: rotate(180deg);
+            }
+          }
+
+          margin-right: 1rem;
+          animation: rotateCarrot 200ms ease-in-out forwards;
+        }
+
+        &--inactive {
+          @keyframes derotateCarrot {
+            0% {
+              transform: rotate(180deg);
+            }
+
+            100% {
+              transform: rotate(0deg);
+            }
+          }
+
+          margin-right: 1rem;
+          animation: derotateCarrot 200ms ease-in-out forwards;
+        }
+      }
+
+      &--dropdown {
+        display: none;
+        margin: 0 -2rem;
+        padding: 0;
+
+        &--active {
```

Comment:
hmm mike's linter doesnt throw an error but this is pretty nested, it should be throwing an error saying this is too nested. ie this is what the class is going to look like: `.nav-mobile__sidebar--dropdown--active`